OpenRA.Game/Primitives/ActionQueue.cs

```diff
@@ -39,7 +37,7 @@ public void PerformActions(long currentTime)
 			DelayedAction[] pendingActions;
 			lock (actions)
 			{
-				var dummyAction = new DelayedAction(null, currentTime);
+				var dummyAction = new DelayedAction(() => { }, currentTime);
```

Comment:
This is the only case where we try to insert a null for the action, so using an no-op lambda seems like a nicer approach.